usr/lib/hypnotix/xtream.py

```diff
@@ -16,16 +16,17 @@
 
 """
 
-__version__ = "0.5.0"
+__version__ = "0.6.0"
 __author__ = "Claudio Olmi"
 
 import json
 import re  # used for URL validation
 import time
-from os import path as osp
 from os import makedirs
+from os import path as osp
+from sys import stdout
 from timeit import default_timer as timer  # Timing xtream json downloads
-from typing import List, Tuple
+from typing import List, Tuple, Protocol
```

Comment:
We need to stay compatible with python 3.9 (Debian Bullseye/LMDE5), It doesn't seem as though Protocol is available there

Reply:
For a moment I though I had something special in my python. Glad it works.